docs/integrations/ai-engines/lightwood.mdx

```diff
@@ -142,6 +142,54 @@ USING
                 ]};
 ```
 
+## Predictions' Explanation
+
+With Lightwood, you can deploy the following types of models:
+
+- regressions models,
+- classification models,
+- time-series models.
+
+Predictions made by each type of model come with an explanation column, as below.
+
+<AccordionGroup>
+
+<Accordion title="Regression">
+In the case of regression models, the `target_explain` column contains the following information:
+
+```
+{"predicted_value": 2951, "confidence": 0.99, "anomaly": null, "truth": null, "confidence_lower_bound": 2795, "confidence_upper_bound": 3107}
+```
+
+Try it out following [this tutorial](/use-cases/in-database_ml/home-rentals).
+</Accordion>
+
+<Accordion title="Classification">
+In the case of classification models, the `target_explain` column contains the following information:
+
+```
+{"predicted_value": "No", "confidence": 0.6629213483146067, "anomaly": null, "truth": null, "probability_class_No": 0.8561, "probability_class_Yes": 0.1439}
+```
+
+<Note>
+The `confidence` score is produced by the conformal prediction module and is statistically valid. On the other hand, the `probability_class` comes directly from the model logits, which may be uncalibrated. Therefore, this `confidence` score is not right, on average - it may be optimistic or pessimistic, for instance.
```

Comment:
Corrections:

- Let's replace "and is statistically valid" by "and is well-calibrated".
- Let's replace the last phrase (starting with "Therefore, this `confidence` score...") by "Therefore, the `probability_class` score may be optimistic or pessimistic, i.e. coverage is not guaranteed to empirically match the reported score."